checks.d/resque.py

```diff
@@ -0,0 +1,36 @@
+# project
+from checks import AgentCheck
+
+import redis
+
+class ResqueVMCheck(AgentCheck):
+
+    SCALAR_KEYS = {
+        'resque:stat:failed': 'resque.jobs_failed',
+        'resque:stat:processed': 'resque.jobs_processed',
+    }
+
+    CARD_KEYS = {
+        'resque:queues': 'resque.queues_count',
+        'resque:workers': 'resque.worker_count',
+    }
+
+    def check(self, instance):
+        tags = instance.get('tags', [])
+
+        host = instance.get('host')
+        port = instance.get('port')
+
+        conn = redis.Redis(host=host, port=port)
```

Comment:
One thing that I have found useful is issuing a service check here if the connection fails.